doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -861,7 +862,7 @@ are applied using the tactic :n:`rewrite_strat @rewstrategy`.
 :n:`@rewstrategy ; @rewstrategy`
    composition
 
-:n:`choice {+ @rewstrategy }`
+:n:`choice {+ @rewstrategy0 }`
```

Comment:
Would you update the nonterminal names in the text (lines 840-958) so the references to `rewstrategy*` exactly match the grammar?  And maybe add some parens in the `choice` construct pseudocode on lines 920-926.

Reply:
Oh, I did not realize all that. Done.

I also tried to use `constr` in place of `glob` in the `mlg` and it seems to work.